### DISCUSSION TITLE
Charge only USB mode

### DISCUSSION THREAD

**Discussion by kanishk619** [gopro/labs#384] [Feature Requests]
Hi

Thanks for the wonderful QR scripting feature for our GoPro's. I wanted to use my gopro as a dashcam in my car but my 12V cigarette lighter always remain powered even after turning off the ignition. However, my infotainment USB ports only works when the ignition is on and gets disabled upon turning the ignition off. 

If I connect the my gopro to the infotainment unit, it goes into USB connection mode. Is there anyway to trigger charge only USB mode and if not can this be taken as a future feature requests ? 

Thanks

**Comment by dnewman-gpsw** [comment]
Which camera model?  

I work-around would to find a charge only cable.  It is a good idea for a Labs feature.

**Comment by kanishk619** [comment]
I am using GoPro Hero 11. I tried finding a charge only cable but no luck.

**Reply by dnewman-gpsw** [reply]
That doesn't make sense, as if it was charge only, it wouldn't have the data lines used to enable the USB connection.

**Reply by kanishk619** [reply]
> That doesn't make sense, as if it was charge only, it wouldn't have the data lines used to enable the USB connection.

I mean I couldn't find a charge only cable sorry for the confusion.

**Reply by tedder** [reply]
Look for "USB power only" or "USB data blocker". They are easy to find.
